arcade/text.py

```diff
@@ -386,7 +386,7 @@ def color(self, color: RGBA255):
         self._label.color = Color.from_iterable(color)
 
     @property
-    def width(self) -> int:
+    def width(self) -> Optional[int]:
```

Comment:
If it is optional, when would we NOT return a width?

Reply:
If you instantiate as `arcade.text.Text(width=None)` then width will be `None`.

https://github.com/pythonarcade/arcade/blob/d7c6986b0241f0304f85a00493e9cd3c51ce184c/arcade/text.py#L185
https://github.com/pythonarcade/arcade/blob/d7c6986b0241f0304f85a00493e9cd3c51ce184c/arcade/text.py#L212